### ISSUE TITLE
[Enhancement]: Import items on the same filesystem as the server from web interface.

### ISSUE THREAD

**Issue by QuickBASIC** [advplyr/audiobookshelf#3400]
### Type of Enhancement

Web Interface/Frontend

### Describe the Feature/Enhancement

I'd like to be able to import items on the server filesystem from the web interface by copying them to the library folder.

### Why would this be helpful?

Audiobooks that have been downloaded (or uploaded)  to the server by other means are often in other folders on the same server as audiobookshelf or audiobooks are being stored in another folder structure elsewhere on the filesystem.

I'd like to be able to import them by copying them to the Library folder from the web interface (hopefully following the folder structure rules after matching metadata with a provider). On a BTRFS filesystem for example these take almost no additional space if copied using a reflink.

### Future Implementation (Screenshot)

![Screenshot](https://github.com/user-attachments/assets/d28a14e4-a8ff-434a-9c44-0181aff8effb)

### Audiobookshelf Server Version

v2.13.4

**Comment by nichwall** [comment]
Is there a reason you can't just add this other media folder to the library?

**Comment by QuickBASIC** [comment]
The intent is that the long term storage of audiobooks for backup is logically separate from the library so that the changes to the files in the audiobookshelf library (embedding metadata or converting to M4B, etc) don't change the original files. The audiobookshelf library is only for audiobookshelf to use, and the original files still existing somewhere else on the filesystem. On modern filesystems that support reflinks there's no additional storage requirements for a copy to be made to the library folder from somewhere else on the filesystem.

Adding the other media folder to audiobookshelf gives audiobookshelf access to those original files and that's what I was trying to avoid.

I'm storing the purchased audiobooks that have had their DRM removed in one directory and I'd like the ability to import those into the existing library folders as a copy. In some cases, purchased audiobooks from some sources have chapter splits and are .mp3 and I would convert them to M4B after import, but still want to retain the original purchased files somewhere else.

Maybe this is incredibly niche and I would totally understand if this isn't implemented and can continue to manually copy things into the library from those other locations. Feel free to close the issue if you feel it's not necessary.

**Comment by QuickBASIC** [comment]
I found a better way with the current functionality to do what I need to do. Making this request superfluous. 

Instead of managing two libraries and needing to import reflink copies to seperate libraries, I started importing all books into a single library and tagging all books as "wifeName" and "myName", and only showing those tags for each user. This means either of us can just remove our tag instead of deleting our reflink copy to remove it from our pseudo library.

Thanks for your time and attention to this, but I'll go ahead and close the Issue since this is a much better solution to my request.
